collective.cpp

```diff
@@ -754,8 +754,9 @@ void Collective::onEvent(const GameEvent& event) {
         if (col->isDiscoverable())
           if (auto enemyId = col->getEnemyId()) {
             if (auto& name = col->getName())
-              control->addMessage(PlayerMessage("The tribe of " + name->full + " is destroyed.",
-                  MessagePriority::CRITICAL));
+              control->addMessage(PlayerMessage(
+                "You draw mana from the destruction of a tribe of " + name->full + ".",
+                MessagePriority::CRITICAL));
```

Comment:
Did you want to replace just the "destroyed" message or also the "surge of power"? Right now it seems you would get two messages about getting mana.

Reply:
This is OK, honest.